src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -62,6 +62,14 @@ const defaultAccessCodeFilter = (
   )
 }
 
+const accessCodeResultToMessage = (
+  result: 'created' | 'updated' | 'deleted' | null
+): string => {
+  if (result === 'created') return t.accessCodeCreated
+  if (result === 'deleted') return t.accessCodeDeleted
+  return t.accessCodeUpdated
+}
+
```

Comment:
Small convention but prefer to move these private functions below (after they are used).